internal/uvm/scsi.go

```diff
@@ -173,17 +187,18 @@ func (uvm *UtilityVM) RemoveSCSI(ctx context.Context, hostPath string) error {
 }
 
 // AddSCSI adds a SCSI disk to a utility VM at the next available location. This
-// function should be called for a RW/scratch layer or a passthrough vhd/vhdx.
-// For read-only layers on LCOW as an alternate to PMEM for large layers, use
-// AddSCSILayer instead.
+// function should be called for a RW/scratch layer, read-only layer as an
```

Comment:
I think this wording is awkward. I would suggest: "This function should be called for adding a scratch layer, a read-only layer as an alternative to VPMEM, or for other VHD mounts.".